Add files via upload

Enzo-Collet opened the pull request:
All good ?

Enzo-Collet committed:
Add files via upload

All good ?

qblanco commented on contact.html:
```diff
@@ -0,0 +1,101 @@
+</div>
+      </div>
+
+     <!-- Right colum w/ most popular location -->
```
why there is the most popular location in the about page ?

qblanco replied:
and in the other pull request Markus already create an about page

Enzo-Collet replied:
Oh yes, my bad, i didn't select the correct file !

qblanco committed:
Update profile.html

Update of profile page
Addition of the agent appartment using the same design as main page
Addition of info form
Next i'm trying to add favorite locations
and an edit page

qblanco committed:
Merge branch 'Hello-There' of https://github.com/IESEG-Java-Web-dev/crispy-pancake into Hello-There

qblanco committed:
Revert "Merge branch 'Hello-There' of https://github.com/IESEG-Java-Web-dev/crispy-pancake into Hello-There"

This reverts commit b97808764f2c7629f6bedc5f53f72320bea6a3db, reversing
changes made to 9b0f85d60d84a11f230c9576f605428022851801.

Enzo-Collet committed:
Add files via upload

Just a little reminder, I was supposed to be the one to do the contact us :).

qblanco committed:
Add

qblanco committed:
Merge branch 'Hello-There' of https://github.com/IESEG-Java-Web-dev/crispy-pancake into Hello-There

qblanco committed:
Revert "Merge branch 'Hello-There' of https://github.com/IESEG-Java-Web-dev/crispy-pancake into Hello-There"

This reverts commit c5472c6e0d0fc3fd09be1597f3027039dabbee14, reversing
changes made to bc97b47fc7f73d7c6ef37eef43304f056590e83c.

qblanco committed:
Revert "Add"

This reverts commit bc97b47fc7f73d7c6ef37eef43304f056590e83c.

qblanco committed:
Revert "Update profile.html"

This reverts commit 9b0f85d60d84a11f230c9576f605428022851801.

qblanco committed:
Revert "Revert "Update profile.html""

This reverts commit bdc76fc4cc69079318854a98659fbd13c848fe47.

qblanco committed:
Revert "Revert "Add""

This reverts commit 235312642252848b72286c87a49ce0f3adeafc0b.
